Stratis.FederatedSidechains.AdminDashboard.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Please remove the solution file, this is single project

Reply:
Ok